IBPSA/Utilities/IO/Reports/CSVWriter.mo

```diff
@@ -5,37 +5,59 @@ model CSVWriter "Model for writing results to a .csv file"
   parameter Integer nin "Number of inputs"
     annotation(Evaluate=true, Dialog(connectorSizing=true));
   parameter String fileName = getInstanceName() + ".csv" "File name, including extension";
-  parameter String delimiter = "\t" "Delimiter for csv file"
-    annotation(Dialog(group="Options"));
   parameter Modelica.SIunits.Time samplePeriod "Sample period";
+  parameter String delimiter = "\t" "Delimiter for csv file"
+    annotation(Dialog(tab="Advanced"));
+
   parameter Boolean writeHeader = true
```

Comment:
It would be nicer to group the `writeHeader` and `headerNames` into a group called `Header`